graylog2-web-interface/src/routing/AppRouter.jsx

```diff
@@ -211,13 +214,17 @@ const AppRouter = () => {
                                component={ShowContentPackPage} />
 
                         <Route exact path={Routes.SYSTEM.GROKPATTERNS} component={GrokPatternsPage} />
-                        <Route exact path={Routes.SYSTEM.INDICES.LIST} component={IndicesPage} />
-                        <Route exact path={Routes.SYSTEM.INDEX_SETS.CREATE} component={IndexSetCreationPage} />
-                        <Route exact path={Routes.SYSTEM.INDEX_SETS.SHOW(':indexSetId')} component={IndexSetPage} />
-                        <Route exact
-                               path={Routes.SYSTEM.INDEX_SETS.CONFIGURATION(':indexSetId')}
-                               component={IndexSetConfigurationPage} />
-                        <Route exact path={Routes.SYSTEM.INDICES.FAILURES} component={IndexerFailuresPage} />
+
+                        {!isCloud && <Route path={Routes.SYSTEM.INDEX_SETS.CREATE} component={IndexSetCreationPage} />}
+                        {!isCloud && (
+                          <Route path={Routes.SYSTEM.INDEX_SETS.SHOW(':indexSetId')} component={IndexSetPage} />
+                        )}
+                        {!isCloud && (
+                          <Route path={Routes.SYSTEM.INDEX_SETS.CONFIGURATION(':indexSetId')}
+                                 component={IndexSetConfigurationPage} />
+                        )}
+                        {!isCloud && <Route path={Routes.SYSTEM.INDICES.LIST} component={IndicesPage} />}
+                        {!isCloud && <Route path={Routes.SYSTEM.INDICES.FAILURES} component={IndexerFailuresPage} />}
```

Comment:
These routes are missing the `exact` prop they had in 4.0. I'm not sure if that was intended or something lost while cherry-picking the changes.

Reply:
That's a good one! I'll add them back